db/migrate/20240903063552_create_orders.rb

```diff
@@ -5,7 +5,7 @@ class CreateOrders < ActiveRecord::Migration[7.1]
   def change
     create_table :orders do |t|
       t.string :order_no
-      t.decimal :total_amount
+      t.float :total_amount
       t.string :invoice
```

Comment:
How do you edit this?? 
Do u create a new migration or down and edit the existing migration???